init.js

```diff
@@ -42,11 +42,11 @@ db.sequelize.sync({ force: true })
     return db.Role.create({ name: 'public' });
   })
   .then(function () {
-    return db.Page.create({title: 'Welcome!', content: 'Welcome to NoBS.js!', slug:'/'})
+    return db.Page.create({title: 'Welcome!', content: 'Welcome to NoBS.js!', slug:'/index'});
```

Comment:
No, I got lazy once I figured out the overall solution. Ill implement this
week

On Sat, Oct 10, 2015 at 6:03 PM, Cody B. Daig <notifications@github.com>
wrote:

> In init.js
> <https://github.com/nobsjs/nobsjs/pull/256#discussion_r41702063>:
>
> > @@ -42,11 +42,11 @@ db.sequelize.sync({ force: true })
> >      return db.Role.create({ name: 'public' });
> >    })
> >    .then(function () {
> > -    return db.Page.create({title: 'Welcome!', content: 'Welcome to NoBS.js!', slug:'/'})
> > +    return db.Page.create({title: 'Welcome!', content: 'Welcome to NoBS.js!', slug:'/index'});
>
> @RRDuke <https://github.com/RRDuke> Did you ever get this added in?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nobsjs/nobsjs/pull/256/files#r41702063>.
>